### PR TITLE
Add test for `setTimeZone`

### PR DESCRIPTION
**Pull request by y-yu**
- Add tests for #77

### CODE REVIEW

File: src/test/scala/com/github/tototoshi/slick/JodaSupportSpec.scala [hunk 1]
```diff
@@ -224,6 +225,41 @@ abstract class JodaSupportSpec extends AnyFunSpec
...
+        Some(new LocalTime(1000))
+      )
+      db.run(jodaTest += data).await()
+
+      val actual = db.run(jodaTest.result).await()
+      actual should have size 1
+
+      actual.foreach { d =>
+        if (isWithoutCalender)
+          d.dateTime.getMillis should be(data.dateTime.getMillis)
```
Comment (y-yu) [inline src/test/scala/com/github/tototoshi/slick/JodaSupportSpec.scala:255]: If `isWithoutCalender == true` then every test will be successful in all timezone.

File: src/test/scala/com/github/tototoshi/slick/JodaSupportSpec.scala [hunk 2]
```diff
@@ -224,6 +225,41 @@ abstract class JodaSupportSpec extends AnyFunSpec
...
+        else if (systemTimeZone == timeZone.getID)
+          d.dateTime.getMillis should be(data.dateTime.getMillis)
```
Comment (y-yu) [inline src/test/scala/com/github/tototoshi/slick/JodaSupportSpec.scala:257]: If `isWithoutCalender == false` and the testing timezone equals to `systemTimeZone` then in/out will be the same.

**Comment by tototoshi** [comment]
👍

**Comment by tototoshi** [comment]
I have no idea why I did thumbs-up but have not merged this

**Comment by y-yu** [comment]
Thanks!

**Pull request merged**